e2e/tests/free5gc/004-free5gc-operator.yaml

```diff
@@ -6,7 +6,7 @@ spec:
   upstream:
     repo: catalog-workloads-free5gc
     package: free5gc-operator
-    revision: main
+    revision: v3.0.0
```

Comment:
This one also fails with the following:

  status:
    conditions:
    - lastTransitionTime: "2024-07-10T22:36:56Z"
      message: could not find upstream package revision 'free5gc-operator/v3.0.0'
        in repo 'catalog-workloads-free5gc'
      reason: UpstreamNotFound